shell/platform/darwin/common/framework/Source/FlutterCodecs.mm

```diff
@@ -78,7 +78,7 @@ - (NSData*)encode:(id)message {
 }
 
 - (id)decode:(NSData*)message {
-  if (message == nil)
+  if (message == nil || [message length] == 0)
```

Comment:
Nit for all of these: you can just do `if ([message length] == 0)` and get the first check for free.

Reply:
Ah good point -- thanks!